packages/sdk/src/services/auth.ts

```diff
@@ -63,13 +63,51 @@ class Auth {
         });
     }
 
+    async isAuthenticated(): Promise<boolean> {
+        const accessToken = this.client.getLocalStorage(
+            "refine-sdk-access-token",
+        );
+
+        if (accessToken) {
+            return true;
+        }
+
+        return false;
```

Comment:
Hey @yildirayunlu, could it like that as well?
```suggestion
   return !!accessToken;
```

Reply:
Thank you for the suggestion @sevkioruc. It can be used. 🚀